CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - FlowClient docs rendered to website now show the options available for arguments that require a string from some set of possibilities [#695](https://github.com/Flowminder/FlowKit/issues/695).
 - The Flowmachine loggers are now initialised only once when flowmachine is imported, with a call to `connect()` only changing the log level [#691](https://github.com/Flowminder/FlowKit/issues/691)
+- The FERNET_KEY environment variable for FlowAuth is now named FLOWAUTH_FERNET_KEY
```

Comment:
Should we mention the `FLOWETL_FERNET_KEY` here as well? (Not sure if it's needed, or if it was already mentioned in a previous PR.)

Reply:
Leave that for a followup? We've no docs on the etl skeleton anyway atm.